### ISSUE TITLE
Recurring Order fields shouldn't be editable unless the Order UI is in edit mode

### ISSUE THREAD

**Issue by michaelrog** [michaelrog/Recurring-Orders#3]
Recurring Order fields shouldn't be editable unless the Order UI is in edit mode.

**Comment by michaelrog** [comment]
(Reached out to P&T for guidance on [detecting/activating 'edit mode'](https://github.com/craftcms/commerce/issues/1904) in the Order Edit app.)

**Comment by michaelrog** [comment]
(Waiting on next Commerce release.)

**Comment by michaelrog** [comment]
(f2377eb is good enough to consider this done for now. Any adjustments/tweaks will get a new issue.)
